src/charset_detect.cpp

```diff
@@ -47,6 +47,8 @@ namespace CharSetDetect {
 
 std::string GetEncoding(agi::fs::path const& filename) {
 	auto encoding = agi::charset::Detect(filename);
+	if (encoding == "ASCII")
```

Comment:
Why?

Reply:
I'd run into cases where UTF-8 files were detected as ASCII. Not entirely sure why, but it should be reliably not a regression, since it's a strict superset.

Reply:
If you have a good repro case you should report that upstream.